internal/rbd/driver/driver.go

```diff
@@ -125,13 +126,17 @@ func (r *Driver) Run(conf *util.Config) {
 			})
 	}
 
-	if conf.EnableReadAffinity {
-		crushLocationMap, err = util.GetCrushLocationMap(conf.CrushLocationLabels, conf.NodeID)
+	if conf.ContainerOrchestrator == "kubernetes" {
```

Comment:
`"kubernetes" ` should be a constant, and if set to something else, a warning needs to be emitted.